AppRater/src/main/java/org/codechimp/apprater/AppRater.java

```diff
@@ -113,13 +131,16 @@ public static void app_launched(Context context) {
      * @param context
      * @param daysUntilPrompt
      * @param launchesUntilPrompt
+     * @param significantEventUntilPrompt
      * @param daysForRemind
      * @param launchesForRemind
+     * @param significantEventForRemind
      */
-    public static void app_launched(Context context, int daysUntilPrompt, int launchesUntilPrompt, int daysForRemind, int launchesForRemind) {

```

Comment:
Replacing methods with the additional parameters will break existing implementations that use this particular calling method.  It would be better to implement your own additional overloaded methods though I'm worried about telescoping this too far and getting a very messy implementation (more than it is today)